lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_generate_strings_file_from_code.rb

```diff
@@ -54,6 +55,11 @@ def self.available_options
                                        description: 'Array of paths or glob patterns to exclude from scanning',
                                        type: Array,
                                        default_value: []),
+          FastlaneCore::ConfigItem.new(key: :routines,
+                                       env_name: 'FL_IOS_GENERATE_STRINGS_FILE_FROM_CODE_ROUTINES',
+                                       description: 'Base name of the alternate methods to be parsed in addition to the standard NSLocalizedString() one',
```

Comment:
Consistency nitpick:

```suggestion
                                       description: 'Base name of the alternate methods to be parsed in addition to the standard `NSLocalizedString()` one',
```